src/simulation/LiveSimulationData.ts

```diff
@@ -32,8 +32,8 @@ const agentB: InputAgent = {
     initialConcentration: 0,
     radius: 1,
     partners: [0],
-    kOn: 0.6,
-    kOff: 0.5,
+    kOn: 0.9,
+    kOff: 0.01,
```

Comment:
these had to be changed because the new unbinding means to have the same binding affinity the k off rate had to be a lot lower